### ISSUE TITLE
Input component warning, use empty string instead of null as default initial empty value.

### ISSUE THREAD

**Issue by petertheprocess** [sopra-fs24-group-09/sopra-fs24-group-09-client#129]
https://github.com/sopra-fs24-group-09/sopra-fs24-group-09-client/blame/e63604985c895cfd73df65cb5eb94d8df19dc56c/src/components/views/Lobby.tsx#L189-L191
```typescript
  const [user, setUser] = useState<User[]>([]); // Is user‘s type User rather than an array: User[]?
  // use empty string instead of null as default initial empty value.
  const [username, setUsername] = useState<string | null>(null);
```
![image](https://github.com/sopra-fs24-group-09/sopra-fs24-group-09-client/assets/30321432/36162d61-b1c8-47bf-8aff-418ec594a9d0)

**Comment by petertheprocess** [comment]
@Haaaan1 @yixuan-zhou-uzh

**Comment by petertheprocess** [comment]
fix on b70735fff81c12fd43cd84bada31cddc898b9eb6

**Comment by yixuan-zhou-uzh** [comment]
the user can now use empty string as a username
